internal/build/buildkit_printer_test.go

```diff
@@ -1,60 +1,104 @@
 package build
 
 import (
+	"fmt"
+	"io/ioutil"
 	"strings"
 	"testing"
 )
 
+const writeGoldenMaster = false
```

Comment:
This is kinda a hacky golden master test. I'm gonna change this const to be a runtime flag that we can pass to change what the golden master is.

Reply:
have you considered doing this with ldflags?

```
go test -ldflags "-X github.com/windmilleng/tilt/internal/build.WriteGoldenMaster=1"
```

should be ok for this use case if we just stick it in a Makefile

Reply:
ooh that's a good idea! I'll set that up tomorrow morning, and maybe abstract this out to be generally usable.